app/views/local_authorities/index.html.erb

```diff
@@ -13,7 +13,13 @@
       locals: {placeholder: "Filter list of local authorities"} %>
     </thead>
     <tbody>
-      <%= render @authorities %>
+      <% @authorities.each do |authority| %>
+        <tr>
+          <td class="name">
+            <%= link_to authority.name, local_authority_path(authority.slug), class: 'js-open-on-submit' %>
```

Comment:
This class lives in the admin template and is so that when pressing enter in the filter box, as some of us have a habit of doing, the first link in the filtered list will open without having to explicitly click it.